arviz/stats/diagnostics.py

```diff
@@ -453,6 +448,12 @@ def mcse(data, *, var_names=None, method="mean", prob=None):
     )
 
 
+@conditional_vect
+def _sqrt(a_a, b_b):
+    return (a_a + b_b) ** 0.5
+
+
+@conditional_jit
 def geweke(ary, first=0.1, last=0.5, intervals=20):
```

Comment:
I have no idea why nor how to solve it, I point it out just in case. If you build the docs locally, you will see that the `geweke` link on the api page is not working anymore. It shows the text, but it does not link to the `geweke` page.

Reply:
Oh I think we might need to do a functools wrap. Decorators "eat" the docstrings of their decorated functions

https://www.thecodeship.com/patterns/guide-to-python-function-decorators/

Reply:
I think we should do this on this PR because we don't want to regress the docs for every function that we use `conditional_jit` for